src/vm/analysis/build_contract_interface/mod.rs

```diff
@@ -1,7 +1,66 @@
+use vm::analysis::types::ContractAnalysis;
```

Comment:
Can we change the name of this module?  `build_contract_interface` sounds like a method name (since it's describing an action).  What if we just called this the `contract_interface` module, or something that is a noun?